src/diapysef/diapysef/data/IMlib/evidence.txt

```diff
@@ -0,0 +1,300 @@
+,Sequence,Length,Modifications,Modified sequence,Oxidation (M) Probabilities,Oxidation (M) Score Diffs,Acetyl (Protein N-term),Oxidation (M),Missed cleavages,Proteins,Leading proteins,Leading razor protein,Gene names,Protein names,Type,Raw file,Experiment,MS/MS m/z,Charge,m/z,Mass,Resolution,Uncalibrated - Calibrated m/z [ppm],Uncalibrated - Calibrated m/z [Da],Mass error [ppm],Mass error [Da],Uncalibrated mass error [ppm],Uncalibrated mass error [Da],Max intensity m/z 0,Retention time,Retention length,Calibrated retention time,Calibrated retention time start,Calibrated retention time finish,Retention time calibration,Ion mobility index,Ion mobility length,Match time difference,Match m/z difference,Match q-value,Match score,Number of data points,Number of scans,Number of isotopic peaks,PIF,Fraction of total spectrum,Base peak fraction,PEP,MS/MS count,MS/MS scan number,Score,Delta score,Combinatorics,Intensity,Reverse,Potential contaminant,id,Protein group IDs,Peptide ID,Mod. peptide ID,MS/MS IDs,Best MS/MS,AIF MS/MS IDs,Oxidation (M) site IDs,IonMobilityIndexK0

```

Comment:
is this file example file?

Reply:
Yes, so there are 2 example MQ libraries. One with ion mobility, and one with regular MQ. Each directory has an evidence and a msms file, but total they are only 1660 kb in size.